testing/trino-faulttolerant-tests/pom.xml

```diff
@@ -14,6 +14,16 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+
+        <!--
+          Project's default for air.test.parallel is 'methods'. By design, 'instances' runs all the methods in the same instance in the same thread,
+          but two methods on two different instances will be running in different threads.
+          As a side effect, it prevents TestNG from initializing multiple test instances upfront, which happens with 'methods'.
+          This reduces tests' memory footprint, which generally allows them to complete faster.
```

Comment:
Any rough estimates on how faster we will complete today ?

Reply:
it's copy over from trino-main, trino-tests, trino-hive, trino-delta-lake and trino-iceberg' poms.
the mileage may vary.